### DISCUSSION TITLE
osx / first run: Docker event stream returned error: Cannot connect to the Docker daemon at...

### DISCUSSION THREAD

**Discussion by meeech** [jesseduffield/lazydocker#459] [Q&A] [answered]
On first run, I was getting the error in the title. Using docker desktop for mac. Docker was running.

**Comment by meeech** [comment, marked as answer]
Solution I found:

the `docker.sock` was not in `/var/run/docker.sock` but in `/Users/<username>/.docker/run/docker.sock` so you can do 

`DOCKER_HOST=unix:///Users/<username>/.docker/run/docker.sock lazydocker`.

if you prefer, it looks like `ln -s` will work as well

`sudo ln -s /Users/<username>/.docker/run/docker.sock /var/run/docker.sock`

**Reply by GabCostaSilva** [reply]
so precise

**Reply by sinhadeepak1115** [reply]
Thanks for the solution.
